srcpkgs/cfm/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'cfm'
+pkgname=cfm
+version=0.6.3
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
```

Comment:
Just set to "yes" here, "compliant" is just different for the sake of differences.

Reply:
I was following what st does. Will update.